.changeset/tame-gifts-tease.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
```

Comment:
Would we classify this as a patch or feature?

It seems like a new feature to me and would result in a new minor version?

I didn't review the substance of the PR but the spirit is awesome. 🏆 

Thank you @rjelierse

Reply:
I was debating whether this should be considered a new function, but new minor function works for me.